test/integration/controllers/leaderworkerset_test.go

```diff
@@ -1512,6 +1511,41 @@ var _ = ginkgo.Describe("LeaderWorkerSet controller", func() {
 				},
 			},
 		}),
+		ginkgo.Entry("create a leaderworkerset with spec.startupPolicyy=LeaderReady", &testCase{
```

Comment:
I think we need a comparative testcase, like when startupPolicy set to leaderCreated, when the leader pods are created, then worker statefulset will be created immediately.